docs/semgrep-cloud-platform/teams.md

```diff
@@ -0,0 +1,298 @@
+
+### Operations permitted per role
+
+| Capability              | Member | Manager | Admin | Notes |
+| ----                    | ------ | ------- | ----- | ----- |
+| Create projects         | ❌ No     | ❌ No      | ✅ Yes   |       |
+| Assign roles            | ❌ No     | ❌ No      | ✅ Yes   |       |
+| Create or edit teams    | ❌ No     | ❌ No      | ✅ Yes   |       |
+| Create or edit subteams | ❌ No     | ✅ Yes     | ✅ Yes   |       |
+| Delete teams            | ❌ No     | ❌ No      | ✅ Yes   |       |
+| Delete subteams         | ❌ No     | ✅ Yes     | ✅ Yes   | A manager can delete the subteams they are a manager of, provided that there are no resources, such as projects, assigned to the subteam.      |
+| API                     | ❌ No     | ❌ No      | ✅ Yes   |       |
+
+## How team access works
+
+- Members of a top-level team gain access to the projects of its subteams. They are indirect members of a subteam.
+- Members of a subteam do not have access to the projects of teams or subteams above it.
+
+In the following diagram, team 1 gains access to subteam 1b's projects, but team 1b does not gain access to projects from team 1.
+
+
+![Team scopes diagram](/img/access-diagram.png#md-width)
+
+- The members Alexis, Pam, and Raj have access to the following projects:
+    - App
+    - Microservices
+    - Frontend
+- The members David, Sebas, and Phaedra have access to the following projects:
+    - Frontend
+
+### The manager role
+
+Use the **manager role** to delegate the assignment of projects across many users. Managers can speed up the deployment of Semgrep into your organization by creating subteams to grant members access to projects.
+
+Given a security engineer who is a manager of **team A** but a member of **team B**, with both teams having the same projects:
+
+- The security engineer has manager **access** to the projects.
+- The security engineer can create subteams for team A but can't create subteams for team B.
+
+Additionally, the manager role is able to perform the following:
+
+- Scan new projects through the **Projects** page.
+- Edit projects that their team is assigned to.
+
+Managers cannot remove themselves from their team. Only admins can remove managers.
+
+![A manager's view of the Projects page.](/img/access-manager-view.png)
+**Figure**. A manager's view of the Projects page. They are able to scan new projects and edit the settings for Projects assigned to Teams they are managers of.
+
+## Enable teams
+
+1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep Cloud Platform](https://semgrep.dev/login).
+1. Click **[<i class="fa-solid fa-gear"></i> Settings > Access > Teams](https://semgrep.dev/orgs/-/settings/access/teams)**.
+1. Click **Enable**.
+
+When you have enabled teams for the first time, a team is automatically created with the name of your deployment. This preserves the settings you previously had using the **Members** feature; All current members retain their existing projects.
```

Comment:
We recently added a feature that optionally allows the default team to also include newly added members. With this option enabled, enabling Teams will preserve the previous behavior (new members will have access to every project in the default team, which we create with all projects). It might be worth highlighting that here.

Reply:
Hmm I don't think there is a figma, we sort of winged it. There's a screenshot in https://github.com/semgrep/semgrep-app/pull/13923